proto/umee/leverage/v1/query.proto

```diff
@@ -135,6 +135,13 @@ service Query {
       returns (QueryMarketSummaryResponse) {
     option (google.api.http).get = "/umee/leverage/v1/market_summary";
   }
+
+  // TotalCollateral returns the total collateral system-wide of a given
+  // uToken denomination.
+  rpc TotalCollateral(QueryTotalCollateralRequest)
+      returns (QueryTotalCollateralResponse) {
+    option (google.api.http).get = "/umee/leverage/v1/total_collateral";
+  }
```

Comment:
Why adding new endpoint? This is already available in `QueryLiquidationTargetsResponse`

Reply:
Query liquidation targets returns a list of _addresses_ whose borrowed value (in USD) is greater than their liquidation threshold.

Reply:
ah, OK :+1: